donate/payments/sqs.py

```diff
@@ -19,13 +23,25 @@ def sqs_client():
 
 
 def send_to_sqs(payload):
+    send_message_retries = 3
     # If BASKET_SQS_QUEUE_URL is not configured, do nothing (djangorq is logging the payload).
     if settings.BASKET_SQS_QUEUE_URL:
         client = sqs_client()
         if client is None:
+            logger.error("Could not connect to SQS Client.")
```

Comment:
I added this because I thought it would be a good idea to let the team know if we cant connect to the client for some reason instead of just skipping with no warning